test/stripe/charge_test.exs

```diff
@@ -3,8 +3,6 @@ defmodule Stripe.ChargeTest do
   use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
   setup do
-    HTTPoison.start
```

Comment:
Spotted this in a few places in test. Not sure what was this for, the tests pass well without this line.